Fix daily command

Discord daily command is broken:

Error in Log: phantombot  | [07-28-2023 @ 09:56:53.326 CEST] [ERROR] [init.js:504] Error with Event Handler [discordChannelCommand] Script [./discord/ice/dailyCommand.js] Stacktrace [setIniDbNumber()@20db.js:160 > dailyCommand.js:116 > init.js:496 > init.js:837] Exception [InternalError: Cannot convert 1690531013323 to java.lang.Integer (20db.js#160)]

--- a/javascript-source/discord/ice/dailyCommand.js
+++ b/javascript-source/discord/ice/dailyCommand.js
@@ -108,12 +108,12 @@
             }
 
             // no subcommand given
-            var userTimestamp = $.getSetIniDbNumber('discordDailyCommandUsers', twitchName, 0),
+            var userTimestamp = parseInt($.getSetIniDbString('discordDailyCommandUsers', twitchName, 0)),
                 currentTime = new Date().getTime(),
                 timeGone = currentTime - userTimestamp,
                 hoursSinceLast = Math.trunc(timeGone / 1000 / 60 / 60);
             if (hoursSinceLast >= 24) {
-                $.setIniDbNumber('discordDailyCommandUsers', twitchName, currentTime);
+                $.setIniDbString('discordDailyCommandUsers', twitchName, currentTime);
                 $.inidb.incr('points', twitchName, payout);
                 $.discord.say(channel, $.discord.userPrefix(mention) + $.lang.get('discord.daily.success', $.getPointsString(payout)));
             } else {

